File: src/extension/compass/__tests__/fixtures/sample_gaps.scala
class GapWidget {
  def render(): Int = 1
}

object GapRunner {
  def runAll(): Int = {
    val widget = new GapWidget()
    val r = widget.render()
    Registry.lookup(r)
    localHelper(r)
  }

  def buildBare(): GapWidget = new GapWidget

  def localHelper(v: Int): Int = v
}

File: src/extension/compass/__tests__/fixtures/sample_gaps.cpp
class GapWidget {
public:
    static int spawnCount();
    int render();
};

int GapWidget::spawnCount() { return 1; }

int GapWidget::render() { return 2; }

int helperFun(int total) { return total; }

int runAll() {
    GapWidget* widget = new GapWidget();
    ns::Tracker* tracker = new ns::Tracker();
    int c = GapWidget::spawnCount();
    int r = widget->render();
    (void)tracker;
    return helperFun(c + r);
}
